.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: python
+language: c
```

Comment:
here be dragons

Reply:
This confirmed that `language: python` was causing failure before my "before_install" commands could run. I suspect that means Travis MacOS builds will explode whenever `language: python`.

I'll give your workflow a shot - hopefully that's a lower pain threshold than attempting to recreate Travis's python environment manually... :dragon: :dragon_face: :dragon: